runtime/debug/strategy-explorer-adapter.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright (c) 2018 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+ 'use strict';
+
+import devtoolsChannelProvider from './devtools-channel-provider.js';
+
+export default class StrategyExplorerAdapter {
+  static processGenerations(generations) {
+    devtoolsChannelProvider.get().send({
+      messageType: 'generations',
+      messageBody: new StrategyExplorerAdapter().adapt(generations)
```

Comment:
I think it would be better if we could `adapt` on the other side of the channel, but I guess that means we need another means to serialize this?

@shans

Reply:
Are you mostly worried about separation of concerns, i.e. so that all the Strategy Explorer logic is encapsulated together?

Reply:
Discussed in person, and Doug's not sure whether introducing additional serialization step is worth it. We decided to proceed as is for now. We can change in the future if we'll want to extend strategy explorer, or use this data on the devtools side for different purpose etc.

Reply:
Add a TODO maybe. I think @dstockwell is right that in the long run we don't want to `adapt()` here. That doesn't need to block merging however.